### PR TITLE
Assignment1

### PR DESCRIPTION
**Pull request by OfirKedem**

### CODE REVIEW

**Comment by kmodan** [comment]
*assignment1/knn.ipynb*
Inline Question 1: please elaborate two cases when test set are very different from all the images in the train set.

*assignment1/svm.ipynb*
please repeat the hyper-parameters tune with different learning rates.
Inline question 2: very good explanation!

*assignment1/cs231n/classifiers/linear_svm.py*
line 124: your mask is already a binary mask. and if it wasn't, this not the proper way to put ones when margin is greater then zero.

*assignment1/cs231n/classifiers/softmax.py*
line 103: loss is calculated for the first time - it should be assignment instead of subtraction
lines 109-111: it can be done in a more efficient way

**Comment by OfirKedem** [comment]
assignment1/cs231n/classifiers/linear_svm.py
line 124: this line just turns the boolian mask to float.

assignment1/cs231n/classifiers/softmax.py
line 103: remark unclear. the loss is initialize to zero at the begining of the function.
